universal-application-tool-0.0.1/app/services/applicant/Block.java

```diff
@@ -88,8 +88,6 @@ public boolean hasErrors() {
   public boolean isCompleteWithoutErrors() {
     // TODO(https://github.com/seattle-uat/civiform/issues/551): Stream only required scalar paths
     // instead of all scalar paths.
-    // TODO: needs to be able to figure out the indices used to reference repeated entities (e.g.
-    // applicant.children[3].name.first).
     return blockDefinition.scalarPaths().stream()
```

Comment:
question: Eventually we'll need to change out of using `blockDefintition#scalarPaths` here, right?

Reply:
yeah this is in the follow-up PR